### ISSUE TITLE
[svelte-parse] unquoted attribute values that include a moustache treated as text

### ISSUE THREAD

**Issue by halfnelson** [pngwn/MDsveX#178]
consider the following
```svelte
<h1 attr=static{dynamic} />
```
svelte parses (and compiles) this as an attribute made of two values (text part and moustache tag)
```json
{
            "type": "Attribute",
            "name": "attr",
            "value": [
              {
                "type": "Text",
                "raw": "static",
                "data": "static"
              },
              {
                "type": "MustacheTag",
                "expression": {
                  "type": "Identifier",
                  "name": "dynamic"
                }
              }
            ]
          }
```

svast currently parses this as
```json
 {
          "type": "svelteProperty",
          "name": "attr",
          "value": [
            {
              "type": "text",
              "value": "staticdynamic",
            }
          ],
          "modifiers": [],
          "shorthand": "none",

        }
```

**Comment by halfnelson** [comment]
However this does work if it is quoted

**Comment by pngwn** [comment]
Ah. This will be due to the special handling of quotes and unquoted attribute values.

**Comment by pngwn** [comment]
This is kindof hardcode to be a single value right now, it needs similar handling to quoted expressions. See https://github.com/pngwn/MDsveX/issues/174#issuecomment-735432870

**Comment by pngwn** [comment]
This should be fixed in `svelte-parse@0.1.0`.
